ha_healthchecker/README.md

```diff
@@ -0,0 +1,146 @@
+# OpenLab HA HealthChecker
+
+OpenLab HA HealthChecker(ha_healthchecker) is a daemon service which is used for monitor and control OpenLab CI environment.
+
+## Concept
+
+First of all, let's introduce some concepts.
+
+1. Master/Slave/Zookeeper
+
+    There are three cluster roles in OpenLab HA development.
+    
+    `Master` is the cluster that OpenLab runs on.
+    
+    `Slave` is the cluster for backup, once master is down, `ha_healthchecker` will switch OpenLab from Master to Slave to ensure OpenLab works. 
```

Comment:
`Slave` is the cluster for backup, once Master is down, `ha_healthchecker` will switch Slave to Master to ensure OpenLab works?

Reply:
Done